--- Makefile.orig	2016-03-21 14:27:35.766923818 -0500
+++ Makefile	2016-03-21 14:27:35.766923818 -0500
@@ -1,5 +1,5 @@
-CC=gcc
-CXX=g++
+CC ?= gcc
+CXX ?= g++
 CFLAGS+=-D_FILE_OFFSET_BITS=64
 #CXXFLAGS+=-Wall -D_FILE_OFFSET_BITS=64 -D USE_UTF16
 CXXFLAGS+=-Wall -D_FILE_OFFSET_BITS=64
@@ -10,6 +10,7 @@
 MBR_LIB_OBJS=$(MBR_LIBS:=.o)
 LIB_HEADERS=$(LIB_NAMES:=.h)
 DEPEND= makedepend $(CXXFLAGS)
+PREFIX ?= /sbin/
 
 all:	cgdisk gdisk sgdisk fixparts
 
@@ -39,6 +40,9 @@
 	$(DEPEND) $(SRCS)
 
 $(OBJS):
-	$(CRITICAL_CXX_FLAGS) 
+	$(CRITICAL_CXX_FLAGS)
 
+install:
+	-mkdir -p $(DESTDIR)/$(PREFIX)
+	cp -a gdisk cgdisk sgdisk fixparts $(DESTDIR)/$(PREFIX)
 # DO NOT DELETE
